src/PKSim.Presentation/Services/SimulationSettingsRetriever.cs

```diff
@@ -58,6 +66,20 @@ public void SynchronizeSettingsIn(Simulation simulation)
          if (simulation.OutputSelections == null) return;
          var clone = simulation.OutputSelections.Clone();
          updateSelectionFromTemplate(simulation, clone);
+         updatePersistable(simulation);
```

Comment:
Once the output selection has been updated, we update the persistable flag in the new simulation

That means that ONLY Brand new simulation without output settings would not be updated. Well I can live with that